components/Player/PlayerPicture.jsx

```diff
@@ -6,33 +6,23 @@ import Avatar from 'material-ui/Avatar';
 import styles from './PlayerHeader.css';
 import { Link } from 'react-router';
 
+//TODO this should be a dumb component
+
 const getPlayerStyle = (registered, cheese) => {
   if (cheese) return styles.playerPictureCheese;
   if (registered) return styles.playerPictureYasp;
   return styles.playerPicture;
 };
 
 const getComponent = (picture, steamLink, cheese, registered, link) => {
-  if (!picture) {
-    return (
-      <Link to={link} className={getPlayerStyle(registered, cheese)}>
-        <FlatButton label="Profile" />
-      </Link>
-    );
-  }
-
-  if (link) {
+  if (picture) {
     return (
-      <Link to={link} className={getPlayerStyle(registered, cheese)}>
-        <Avatar src={picture} />
+      <Link to={link || steamLink} className={getPlayerStyle(registered, cheese)}>
+        <img src={picture} />
       </Link>
     );
   }
-  return (
-    <a href={steamLink} className={getPlayerStyle(registered, cheese)}>
-      <Avatar src={picture} />
-    </a>
-  );
+  return <div />;
 };
```

Comment:
Ok, I don't really care either way about the square vs circle, but I don think that circlular profile pictures is something a lot of modern apps use (gchat/hangouts in particular). I think hard edges looks strange, and slightly rounded edges doesn't really look great either. I don't really care either way though I guess.